lib/pages/main/wallet_contents/explorer/explorer_result_page.dart

```diff
@@ -218,12 +218,12 @@ class _ExplorerResultPageState extends State<ExplorerResultPage> {
 
           if (widget.resultType == ExplorerResultType.publicId) {
             explorerUrl += "/network/address/$qubicId";
-          } else if (widget.resultType == ExplorerResultType.tick) {
```

Comment:
I think there are some missing braces, the code doesn't compile for me.

Reply:
fixed